src/stylus/components/_alerts.styl

```diff
@@ -17,6 +17,8 @@
     align-self: center
     color: $alert-icon-color
     font-size: $alert-icon-font-size
+  &--outline .icon
+    color: currentColor !important
```

Comment:
Can this be set to just inherit the color and ditch the **!important**?

Reply:
I'll check and confirm, but from what I remember !important was needed. By inherit do you mean still using `currentColor` or something like `color: inherit`?

Reply:
Either if they will work without important. If it's needed though, just use `color: inherit`

Reply:
Both and `inherit` and `currentColor` don't work without `!Important` (they appear grey). Changing this to use `color: inherit !Important` then.